src/web/app/components/session-edit-form/time-picker/time-picker.component.html

```diff
@@ -2,7 +2,7 @@
         (ngModelChange)="triggerTimeChange($event)" [disabled]="isDisabled"
         [compareWith]="timeCompareFn"
 >
-  <option *ngFor="let i of range(23)" [ngValue]="{ hour: i, minute: 0 }">
+  <option *ngFor="let i of range(1, 23)" [ngValue]="{ hour: i, minute: 0 }">
```

Comment:
There's a pre-existing bug here that we need to fix; the range should run from `1-24` instead of `1-23`